app/jobs/datadog_sync_job.rb

```diff
@@ -2,29 +2,25 @@ class DatadogSyncJob < ApplicationJob
   queue_as :default
 
   def perform(job)
-      conn = Faraday.new(
-        url: "https://api.datadoghq.com",
+      datadog_credential = Datadog.find(job[:id])
+      credential = Cred.find_by(credable_id: job[:id])
+      response = DatadogServices::ListUsers.new.call(
+        api_key: datadog_credential.api_key,
+        application_key: datadog_credential.application_key,
+        subdomain: datadog_credential.subdomain,
```

Comment:
You can pass datadog_credential object to the list user constructor, then use it from within instead of passing in method arguments